### DISCUSSION TITLE
(필독!!!) 쿠링 iOS 스위프트 스타일 가이드

### DISCUSSION THREAD

**Discussion by x-0o0** [KU-Stacks/kuring-lite-ios#9] [Announcements]
# 코드 스타일 가이드 | The Style Guidelines

모두가 다같이 참여할 수 있는 오픈 소스 프로젝트이기 때문에 가독성과 코드 이해도를 높이기 위해 다음의 스타일가이드에 따라서 코드를 작성해주세요.

[👉🏼 쿠링iOS 스타일가이드: 노션페이지에서 확인하기](https://www.notion.so/kuring/iOS-273caae65c484d6794481d5fb5a96d1d)

**Comment by lgvv** [comment]
이 페이지는 따로 깃헙으로 빼서 스타일쉐어 가이드처럼 Kuring-SwiftUI-StyleGuide를 최초로 만들면 어떨까요 🙌🏻
